Datasets/LanguageModelDataset.swift

```diff
@@ -1,65 +1,63 @@
 import TensorFlow
 
-//Build a dataset suitable for language modeling from an array of texts
-public struct LanguageModelDataset<Item> {
-  public typealias Index = Int
-  public typealias Element = TensorPair<Int32, Int32>
-
-  //A function that reads Item to get an array of Int
-  public let openItem: (Item) -> [Int]
-  //The size of a batch
+/// A dataset suitable for language modeling.
+///
+/// - Note: This struct does not handle the preprocessing required in NLP
+/// and expects you have already tokenized and numericalized your raw texts
+/// (that is split them in tokens, then mapped those tokens to their ids in your
+/// vocabulary). Therefore the generic type Texts refers to a collection of
```

Comment:
Please take care to add backticks! It can really speed up review if you proofread your writing or write backticks the first time :)
```suggestion
/// vocabulary). Therefore the generic type `Texts` refers to a collection of
```

Reply:
I actually wasn't sure on this one since it's not a type but a generic parameter. Sorry about that!